app/pages/organizations/organization-page.jsx

```diff
@@ -0,0 +1,245 @@
+import React from 'react';
+import classnames from 'classnames';
+import { Markdown } from 'markdownz';
+import Translate from 'react-translate-component';
+import Thumbnail from '../../components/thumbnail';
+import ProjectCard from '../../partials/project-card';
+
+const AVATAR_SIZE = 100;
+
+export const OrganizationProjectCards = ({ errorFetchingProjects, fetchingProjects, projects }) => {
+  if (fetchingProjects) {
+    return (
+      <div className="organization-page__projects-status">
+        <p><Translate content="organization.home.projects.loading" /></p>
+      </div>);
+  } else if (errorFetchingProjects) {
+    return (
+      <div className="organization-page__projects-status">
+        <p><Translate content="organization.home.projects.error" /></p>
+        <p>
+          <code>{errorFetchingProjects.toString()}</code>
+        </p>
+      </div>);
+  } else if (!fetchingProjects && !projects.length) {
+    return (
+      <div className="organization-page__projects-status">
+        <p><Translate content="organization.home.projects.none" /></p>
+      </div>);
+  } else {
+    return (
+      <div className="project-card-list">
+        {projects.map(project =>
+          <ProjectCard key={project.id} project={project} />
+        )}
+      </div>);
+  }
+};
+
+OrganizationProjectCards.propTypes = {
+  errorFetchingProjects: React.PropTypes.shape({
+    message: React.PropTypes.string
+  }),
+  fetchingProjects: React.PropTypes.bool,
+  projects: React.PropTypes.arrayOf(
+    React.PropTypes.shape({
+      id: React.PropTypes.string,
+      display_name: React.PropTypes.string
+    })
+  )
+};
+
+class OrganizationPage extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      readMore: false
+    };
+  }
+
+  toggleReadMore() {
+    this.setState({ readMore: !this.state.readMore });
+  }
+
+  render() {
+    const [aboutPage] = this.props.organizationPages.filter(page => page.url_key === 'about');
+
+    const aboutContentClass = classnames(
+      'organization-details__about-content',
+      { 'organization-details__about-content--expanded': this.state.readMore });
+
+    return (
+      <div className="organization-page">
+        <section
+          className="organization-hero"
+          style={{ backgroundImage: `url(${this.props.organizationBackground.src})` }}
```

Comment:
Not a blocker, but something to think about with the style pass. This will probably error if the organization doesn't have a background.

Reply:
great catch! I'll improve in next iteration if ok? the avatar has a fallback stripey look, background has gradient, https://org-landing-page.pfe-preview.zooniverse.org/organizations/markb-panoptes/noavnoback, definitely could use some refinement.

Reply:
Yeah totally ok to fix this with the style fixes.